enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -374,3 +343,41 @@ Provide a recipe for customer's to deploy their own log collector to move the re
 
 ## Infrastructure Needed
 * Future target endpoints may require special infrastructure or licencing to properly test.  For example, Splunk is destination requested by many users but is a licensed product.  The free dev tier may not provide sufficient capacity to fully test
+* Scale and stress tests require intensive use of a large cluster for an extended period of time.
+## Future plans
+
+As well as serving the current GA requirements, the log forwarding API has been designed with the following future requirements in mind.
+
+### Stand-alone log forwarding. ###
```

Comment:
+1 on this, for an OCP installation shipping logs off cluster it's hard to justify deploying Elastic and Kibana

Reply:
We may not get there for GA, but I'd like to if we can.  For GA we might leave a "special path" to our elastic stuff, but in the very near future we want the forwarder to be forwarding to elastic as just another destination.